Loop/Managers/Alerts/StoredAlert.swift

```diff
@@ -145,6 +149,48 @@ extension Alert.Trigger {
     }
 }
 
+extension Alert.InterruptionLevel {
+    enum StorageError: Error {
+        case invalidStoredLevel
+    }
+    
+    var storedValue: Int16 {
+        // Since this is arbitrary anyway, might as well make it match iOS's values
+        switch self {
+        case .active:
+            if #available(iOS 15.0, *) {
+                return Int16(UNNotificationInterruptionLevel.active.rawValue)
+            } else {
+                return 1
```

Comment:
[nit] Might want some indication that `UNNotificationInterruptionLevel.active.raw` **IS** `1`. My first thought was "is there a problem when upgrading from iOS 14 with a value of `1` to iOS 15 with a value of `Int16(UNNotificationInterruptionLevel.active.rawValue)`"?

Reply:
I'll put a link to the Apple docs.